plugins/jobtype/src/azkaban/jobtype/pig/PigUtil.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.jobtype.pig;
+
+import java.util.Iterator;
+import java.util.Set;
+
+import org.apache.hadoop.mapred.Counters;
+import org.apache.pig.PigRunner;
+import org.apache.pig.tools.pigstats.JobStats;
+import org.apache.pig.tools.pigstats.PigStats;
+import org.apache.pig.tools.pigstats.PigStats.JobGraph;
+
+import azkaban.jobtype.AzkabanPigListener;
+import azkaban.utils.Props;
+
+
+public class PigUtil {
```

Comment:
Can we add Javadoc for this class and the two undocumented methods?

Reply:
Done.